interfaces/builtin/network_control.go

```diff
@@ -280,6 +284,43 @@ var networkControlConnectedPlugUDev = []string{
 	`KERNEL=="tun"`,
 }
 
+var networkControlConnectedPlugMount = []osutil.MountEntry{{
+	Name:    "/var/lib/snapd/hostfs/var/lib/dhcp",
+	Dir:     "/var/lib/dhcp",
+	Options: []string{"bind", "rw", osutil.XSnapdIgnoreMissing()},
+}}
+
+// TODO: Add a layer that derives this sort of data from mount entry, like the
+// one above, into a set of apparmor rules for snap-update-ns, like the ones
+// below.
+//
+// When setting up a mount entry, we also need corresponding
+// snap-updates-ns rules. Eg, if have:
+// []osutil.MountEntry{{
+//	Name:    "/foo/bar",
+//	Dir:     "/bar",
+//	Options: []string{"rw", "bind"},
+// }}
+// Then you can expect to need:
+// /foo/ r,
+// /foo/bar/ r,
+// mount options=(rw bind) /foo/bar/ -> /bar/,
+// umount /bar/,
+// ...
+// You'll need 'rw' rules to support cases when ...
```

Comment:
@zyga - the '...' part of  'to support cases when ...' part was meant to be filled in by you. :)

Reply:
Oh well. That would be too easy :-)

Reply:
Done